docs/static/releasenotes.asciidoc

```diff
@@ -13,6 +14,69 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-2-1]]
+=== Logstash 8.2.1 Release Notes
+
+[[notable-8.2.1]]
+==== Notable issues fixed
+
+* Added mandatory JVM option to avoid strict path checking introduced with recent JVM versions,
+  starting from 11.0.15+10, 17.0.3+7.
+https://github.com/elastic/logstash/pull/14066[#14066]
+
+* Fixed Dead Letter Queue bug happening in position retrieval and restore. This happened when the DLQ input plugin used
+  `commit_offset` feature.
+https://github.com/elastic/logstash/pull/14093[#14093]
+
+* Fixes an issue where custom java plugins were unable to be installed and run correctly when retrieved from rubygems.org.
+https://github.com/elastic/logstash/pull/14060[#14060]
+
+* Fixed no metrics update issue when PQ is draining.
+https://github.com/elastic/logstash/pull/13935[#13935]
+
+==== Plugins
+
+*Cef Codec - 6.2.5*
+
+* [DOC] Update link to CEF implementation guide https://github.com/logstash-plugins/logstash-codec-cef/pull/97[#97]
+
+*Dns Filter - 3.1.5*
+
+* Fixed an issue where a non-string value existing in the resolve/reverse field could cause the plugin to crash
```

Comment:
```suggestion
* Fixed an issue where a non-string value existing in the resolve/reverse field could cause the plugin to crash https://github.com/logstash-plugins/logstash-filter-dns/pull/65[#65]
```

Reply:
Thanks for handling this release. @andsel !

Please merge this PR (or delegate merging to me) the day before the release to help facilitate a smooth docs release. The docs release manager for this release, @colleenmcginnis, will thank you. 😸